tutorials/2_sprite_animations_gestures/README.md

```diff
@@ -0,0 +1,241 @@
+# Basic: Sprites, Animations and Gestures
+
+This tutorial will introduce you to:
+
+ - `Sprite`: Sprites are how we draw images, or portions of an image in Flame.
+ - `SpriteAnimation`: SpriteAnimations are animations composed from sprites, where each sprite represents a frame.
+ - Gesture input: Input is a crucial part of game development, on this tutorial you will learn how to use one of the most basic of them, the tap detector.
+
+All the code of this tutorial code can be found [here](./code) and is based on the Flame `1.0.0-rc8` version.
+
+By the end of this tutorial you will have built a simple game which renders a button which when pressed, makes a small vampire robot run, that will look like this:
+
+![Preview](./media/preview.gif)
+
+## Sprite
+
+Before starting coding our game, it is important to understand what sprites are and what they are used for.
+
+Sprites are images, or a portion of an image loaded into the memory, which can then be used to render graphics on your game canvas.
+
+Sprites can be (and usually are) bundled into single images, that is a very useful technique as it lowers the amount of I/O operations needed to load the game assets because it is faster to load 1 image of 10k than to load 10 images of 1kb each.
```

Comment:
```suggestion
Sprites can be (and usually are) bundled into single images, called Sprite Sheets. That is a very useful technique as it lowers the amount of I/O operations needed to load the game assets because it is faster to load 1 image of 10 KB than to load 10 images of 1 KB each (among other advantages).
```

I think we should link an article with more details on the rationale behind spritesheets at the end of this sentence, like: https://gamedev.stackexchange.com/questions/7069/2d-graphics-why-use-spritesheets (or something better.

I think the biggest advantages are (1) just organization and (2) batch GPU calls more effectively (I think this is more important than the IO performance).

Reply:
I like the idea of linking an article, any suggestion? I would rather link a real article instead of question on a forum like that one

Reply:
yeah me too but I didn't easily found one (which I thought I would). might require some deeper digging, there must certainly be one haha